sqlmesh/core/context.py

```diff
@@ -205,22 +200,30 @@ def __init__(
         users: t.Optional[t.List[User]] = None,
     ):
         self.console = console or get_console()
-        self.path = Path(path).absolute()
-        if not self.path.is_dir():
-            raise ConfigError(f"{path} is not a directory")
 
-        self.config = self._load_config(config or "config")
+        self._sqlmesh_path = Path.home() / ".sqlmesh"
 
-        self.physical_schema = physical_schema or self.config.physical_schema or "sqlmesh"
-        self.snapshot_ttl = snapshot_ttl or self.config.snapshot_ttl or c.DEFAULT_SNAPSHOT_TTL
-        self.dag: DAG[str] = DAG()
+        _paths = [
+            Path(path).absolute() for path in ([paths] if isinstance(paths, str) else list(paths))
+        ]
+
+        for path in _paths:
+            if not path.is_dir():
+                raise ConfigError(f"{path} is not a directory")
```

Comment:
Do we even need this check? I'm pretty sure we're going to check for whether the configuration file exists, so should we just check that instead?

Reply:
i'll get rid of it

Reply:
Or otherwise shouldn't this be a part of `_load_configs`?